pkg/channel/consolidated/dispatcher/consumer_message_handler.go

```diff
@@ -70,6 +71,8 @@ func (c consumerMessageHandler) Handle(ctx context.Context, consumerMessage *sar
 		zap.String("subscription", c.sub.String()),
 	)
 
+	httpHeader := tracing.ConvertRecordHeadersToHttpHeader(tracing.FilterCeRecordHeaders(consumerMessage.Headers))
```

Comment:
wanna add a comment here similar to the other invocation?

Reply:
sure - you bet!